tests/e2e/components/demo-modal-compare.spec.ts

```diff
@@ -99,25 +99,25 @@ const modalTestConfigs: ModalTestConfig[] = [
 const componentName = 'Modal';
 
 const testConfigs: TestConfig[] = [
-  {
```

Comment:
⚠️ Please, add a comment explaining why the code is commented out (ideally with the date and author) and when it will be enabled again (ideally with an issue link, if we can have it). Forgetting it here as it is could disappoint anyone who opens this code in the future 🙂.